src/scrape_up/finance/stock_price.py

```diff
@@ -76,5 +76,3 @@ def get_historical_data(self, from_date, to_date):
             }
 
 
-sp = StockPrice(stock_index="infosys")
-sp.get_historical_data()
```

Comment:
Same in this case also 
![image](https://github.com/Clueless-Community/scrape-up/assets/93156825/71a19dc9-a4cc-43e0-ae90-a62737d83522)

Reply:
> ![image](https://user-images.githubusercontent.com/93156825/244474234-71a19dc9-a4cc-43e0-ae90-a62737d83522.png)

The stock_price.py is not working because in nse.py you already changed the the stock name to "RELIANCE INDUSTRIES LTD" (which is a special case) and ```get_historical_data``` take from and to dates as input. The StockPrice class used nse class to get results and the error would pop up as soon as the nse.py is imported. If you want to test anything please do that by either importing the stock_price.py or put the testing code in 
```py
if __name__=="__main__":
```
So the code wont execute while being imported(only executes when directly run). Please try the first method and avoid directly.
The methods are working in my fork try using them by creating a test.py file in finance module with following code:
```py
from stock_price import StockPrice

infosys = StockPrice('infosys')
print((infosys.get_historical_data('02-05-2023', '31-05-2023')))
```
if StockPrice works that means NSE is working as well.

Reply:
here's the screenshot of running the code:
![Screenshot from 2023-06-09 07-30-05](https://github.com/Clueless-Community/scrape-up/assets/71490015/558d3160-7316-468f-abcd-5084e9efec03)